<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCImageMainWindow</class>
 <widget class="QMainWindow" name="CCImageMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CCImageMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="browsing_widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3">
       <item>
        <widget class="QWidget" name="view_preview_now_widget" native="true">
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="PreviewLabel" name="label_prev">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>120</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PreviewLabel" name="label_current">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>120</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="PreviewLabel" name="label_next">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>140</width>
              <height>120</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="DisplayLabel" name="label_display">
         <property name="sizePolicy">
          <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>460</width>
           <height>390</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>双击我添加图片</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignmentFlag::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="operating_widget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>645</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonStyle::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_add_images"/>
   <addaction name="action_loadfromDir"/>
   <addaction name="action_prev_image"/>
   <addaction name="action_next_image"/>
  </widget>
  <action name="action_add_images">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::DocumentNew" resource="../../Sources/sources.qrc">
     <normaloff>:/icons/toolbar/icons/toolbar/importImages.png</normaloff>:/icons/toolbar/icons/toolbar/importImages.png</iconset>
   </property>
   <property name="text">
    <string>添加多张图像</string>
   </property>
   <property name="toolTip">
    <string>添加多张图像</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_loadfromDir">
   <property name="icon">
    <iconset theme="QIcon::ThemeIcon::FolderNew" resource="../../Sources/sources.qrc">
     <normaloff>:/icons/toolbar/icons/toolbar/importFolder.png</normaloff>:/icons/toolbar/icons/toolbar/importFolder.png</iconset>
   </property>
   <property name="text">
    <string>添加文件夹图像</string>
   </property>
   <property name="toolTip">
    <string>从文件夹选取图像添加</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_next_image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/toolbar/icons/toolbar/next.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>下一张图像</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
  <action name="action_prev_image">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/toolbar/icons/toolbar/prev.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>上一张图像</string>
   </property>
   <property name="menuRole">
    <enum>QAction::MenuRole::NoRole</enum>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DisplayLabel</class>
   <extends>QLabel</extends>
   <header>DisplayLabel.h</header>
  </customwidget>
  <customwidget>
   <class>PreviewLabel</class>
   <extends>QLabel</extends>
   <header>PreviewLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Sources/sources.qrc"/>
 </resources>
 <connections/>
</ui>
